packages/@aws-cdk/aws-lambda/lib/layer-version.ts

```diff
@@ -0,0 +1,163 @@
+import cdk = require('@aws-cdk/cdk');
```

Comment:
I would just call this file “layers.ts” as it includes a few things related to layers